apps/omg_watcher/config/config.exs

```diff
@@ -13,7 +13,8 @@ config :omg_watcher,
   maximum_block_withholding_time_ms: 10_000,
   block_getter_height_sync_interval_ms: 2_000,
   maximum_number_of_unapplied_blocks: 50,
-  eth_exit_finality_margin: 12
+  eth_exit_finality_margin: 12,
+  exit_processor_validation_interval_ms: 5_000
```

Comment:
add a shorter version of this interval to `config/dev` and even a shorter one to `config/test`